diff -Naur qtermwidget-0.14.1/lib/TerminalCharacterDecoder.cpp qtermwidget-0.14.1.tpg/lib/TerminalCharacterDecoder.cpp
--- qtermwidget-0.14.1/lib/TerminalCharacterDecoder.cpp	2019-02-25 22:13:12.000000000 +0000
+++ qtermwidget-0.14.1.tpg/lib/TerminalCharacterDecoder.cpp	2020-03-10 09:40:58.712658123 +0000
@@ -31,6 +31,8 @@
 // Konsole
 #include "konsole_wcwidth.h"
 
+#include <cwctype>
+
 using namespace Konsole;
 PlainTextDecoder::PlainTextDecoder()
  : _output(0)
